app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,55 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs {
+  @include media-down(mobile) {
```

Comment:
I know this is WIP so not a review, merely suggestion to use `govuk-media-query` mixin

Reply:
good idea...will do!